doc/radosgw/config-ref.rst

```diff
@@ -1115,6 +1115,45 @@ Keystone Settings
 :Type: Boolean
 :Default: ``true``
 
+
+
+Object Lifecycle Management Settings
+====================================
+
+
+``rgw enable lc threads``
+
+:Description: Whether the Ceph Object Gateway lifecycle management is enabled.
+:Type: Boolean
+:Default: ``true``
+
+
+``rgw lc lock max time``
+
+:Description: The total run time in seconds for a single lifecycle processor.
+:Type: Integer
+:Default: 60
+
+
+``rgw lc max objs``
+
+:Description: The maximum number of objects that may be handled by
+              lifecycle management thread in one processing cycle.
+              Shoud not be larger than 7877(HASH_PRIME).
```

Comment:
@mikulely the description here is incorrect. That's the max number of rados objects that hold the list of objects that are destined for trimming.